libraries/Microsoft.Bot.Schema/Activity.cs

```diff
@@ -1,23 +1,51 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Bot.Schema
 {
     /// <summary>
     /// An Activity is the basic communication type for the Bot Framework 3.0
     /// protocol.
     /// </summary>
-    public partial class Activity
+    [DebuggerDisplay("[{Type}] {Text ?? System.String.Empty}")]
+    public class Activity :
+        IActivity,
+        IConversationUpdateActivity,
+        IContactRelationUpdateActivity,
+        IInstallationUpdateActivity,
+        IMessageActivity,
+        IMessageUpdateActivity,
+        IMessageDeleteActivity,
+        IMessageReactionActivity,
+        ISuggestionActivity,
+        ITypingActivity,
+        IEndOfConversationActivity,
+        IEventActivity,
+        IInvokeActivity,
+        ITraceActivity,
+        IHandoffActivity,
+        ICommandActivity,
+        ICommandResultActivity
     {
+        /// <summary>
+        /// The HTTP <c>Content-Type</c> entity header that identifies an <see cref="Activity"/> media type resource.
+        /// </summary>
+        /// <remarks>In multi-part HTTP content, this header identifies the activity portion of the content.</remarks>
+        public const string ContentType = "application/vnd.microsoft.activity";
```

Comment:
Shouldn't this be inside the ContentType class o "nada que ver"?

Reply:
No, the ContentType class includes only the cards' content types. I don't think this belongs there.